packages/story-editor/src/utils/isOffCanvas.ts

```diff
@@ -21,8 +21,9 @@ import {
   PAGE_HEIGHT,
   PAGE_WIDTH,
 } from '@googleforcreators/units';
+import type { ElementBox } from '@googleforcreators/elements';
```

Comment:
`ElementBox` is in the `units` package, not the `elements` package. You should see a warning about this in your IDE and by TS.

```suggestion
import type { ElementBox } from '@googleforcreators/units';
```